conjure-java-core/src/test/java/com/palantir/conjure/java/JerseyEteResource.java

```diff
@@ -179,6 +179,12 @@ public String jsonErrorsHeader(AuthHeader authHeader, String _headerParameter) {
         return "hello!";
     }
 
+    @Override
+    public String errorParameterSerialization(AuthHeader authHeader, String headerParameter) {
+        // Conjure-Java has not supported Jersey for a while. We are not testing the error serialization feature here.
+        return "hello!";
```

Comment:
nit: should we just explicitly throw to indicate that this is not meant to be tested? (that way, we'd catch more easily that this endpoint somehow ended up in a test somewhere)